cmd/runtimetest/main.go

```diff
@@ -11,6 +11,7 @@ import (
 	"syscall"
 
 	"github.com/Sirupsen/logrus"
+	"github.com/docker/docker/pkg/mount"
```

Comment:
> but I worry about cyclic dependencies 
Make sense.

> Can generic mount parsing/handling get pulled out into an independent repository somewhere so that it's clear that it's also low-level (if it is, I haven't looked)?
I'll search that today. Or I'll pull the mount function to ocitools.